### DISCUSSION TITLE
Please can we have a web3 tag so I can block it?

### DISCUSSION THREAD

**Discussion by peterjcole** [dailydotdev/daily#517] [Feature Requests]
It's good that we can block #blockchain but many web3 articles aren't tagged as such, e.g. https://app.daily.dev/posts/NmpWW6uGI

Can we have a new tag for web3 as I don't want to read about it?
 
Thanks!

**Comment by idoshamun** [comment]
Hi @peterjcole,

Your request makes total sense, we're working on rolling a new tagging system.
It will take a while but it should support web3 and others. Stay tuned :)
